File: Core/Entities/Article.cs
using System;

namespace Core.Entities
{
    public class Article : BaseEntity
    {
        public DateTime Date { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Text { get; set; }
        public string PictureUrl { get; set; }
        public string GalleryUrls { get; set; }
        public ArticleCategory ArticleCategory { get; set; }
        public int ArticleCategoryId { get; set; }
    }
}

File: client/src/app/shared/models/article.ts
export interface IArticle {
  id: number;
  date: Date;
  title: string;
  description: string;
  text: string;
  pictureUrl: string;
  galleryUrls: string[];
  articleCategory: string;
}

export interface IArticleAdd {
  date: Date;
  title: string;
  description: string;
  text: string;
  pictureUrl: string;
  galleryUrls: string[];
  articleCategoryId: number;
}

export interface IArticleEdit {
  id: number;
  date: Date;
  title: string;
  description: string;
  text: string;
  pictureUrl: string;
  galleryUrls: string[];
  articleCategoryId: string;
}


